Dockerfile

```diff
@@ -51,9 +76,9 @@ RUN pip install --no-cache-dir --upgrade pshtt==0.6.6
 # Install domain-scan
 ###
 RUN git clone https://github.com/18F/domain-scan \
```

Comment:
This gets us only what we need from the `18F/domain-scan` repository. I added a comment to explain the use of the `SHELL` command.

```suggestion
#
# The SHELL command is used to ensure that if either the curl call or
# the tar call fail then the image build fails. Source:
# https://github.com/docker/docs/blob/main/develop/develop-images/dockerfile_best-practices.md#using-pipes
###
SHELL ["/bin/bash", "-o", "pipefail", "-c"]
RUN mkdir ${CISA_HOME}/domain-scan \
    && curl -L https://github.com/18F/domain-scan/tarball/master \
    | tar zx --strip-components 1 -C ${CISA_HOME}/domain-scan/
```

Reply:
Are you OK with this slightly modified version?
```suggestion
#
# The SHELL command is used to ensure that if either the curl call or
# the tar call fail then the image build fails. Source:
# https://github.com/docker/docs/blob/main/develop/develop-images/dockerfile_best-practices.md#using-pipes
###
SHELL ["/bin/bash", "-o", "pipefail", "-c"]
RUN mkdir ${CISA_HOME}/domain-scan \
    && curl --location https://github.com/18F/domain-scan/tarball/master \
    | tar --extract --gzip --strip-components 1 --directory ${CISA_HOME}/domain-scan/
```

Reply:
Also, what is the difference between `bash -o` and `bash -O`?  The latter is explicitly described in the `bash` `man` page, while the former is not.

Reply:
> All of the single-character shell options documented in the
description of the `set` builtin command, including `-o`, can be used
as options when the shell is invoked.

So in this case we are indirectly doing `set -o pipefail` like we would do for most of our bash scripts.

Reply:
Regarding your modified changes yes I'm sorry I tested with short options just out of convenience and forgot to update when I made my suggestion. I much prefer your version with long options to correctly document what we're doing.